build_sysext

```diff
@@ -27,6 +27,14 @@ DEFINE_string image_builddir '' \
   "Custom directory to build the sysext in. Defaults to a 'sysext' sub-directory of the directory the squashfs base image resides in; '${default_imagedir}/sysext' by default."
 DEFINE_string manglefs_script '' \
   "A path to executable that will customize the rootfs of the sysext image."
+DEFINE_boolean generate_pkginfo "${FLAGS_FALSE}" \
+  "Generate an additional squashfs '<sysext_name>_pkginfo.raw' with portage package meta-information (/var/db ...). Useful for creating sysext dependencies; see 'base_pkginfo' below."
+DEFINE_string base_pkginfo "" \
+  "Colon-separated list of pkginfo squashfs paths / files generated via 'generate_pkginfo' to base this sysext on. The corresponding base sysexts are expected to be merged with the sysext generated."
+DEFINE_string compression "zstd" \
```

Comment:
```suggestion
DEFINE_string compression "gzip" \
```
It's easier to merge the PR if we switch to `zstd` as default in a follow-up PR that we merge after the next round of releases. Then we don't have to build kernel modules for the Jenkins workers.

Reply:
Since the releases already got delayed and there were some concerns regarding release member availability this week, I would say that choosing a fast and easy route is the best.

Reply:
That's OK, Jenkins workers have been updated with a custom kmod already: https://github.com/flatcar/flatcar-linux-infra/pull/268. The kmod will automatically be rebuilt when Jenkins nodes update the OS. The custom kmod service won't run for releases higher than 3760. I've set a reminder for early December to clean up the few artifacts.

I'll start a new build and will update the summary of this PR accordingly.